README.md

```diff
@@ -9,6 +9,12 @@ In this GitHub repository, the following SuiteCloud SDK tools are available:
     <img src="https://img.shields.io/npm/v/@oracle/suitecloud-cli.svg" alt="npm-cli-nodejs"/>
 </a>
 
+- [SuiteCloud Extension for Visual Studio Code](./packages/vscode-extension)
+<a href="https://marketplace.visualstudio.com/items?itemName=Oracle.suitecloud-vscode-extension">
+    <img src="https://img.shields.io/visual-studio-marketplace/i/oracle.suitecloud-vscode-extension.svg" alt="suitecloud-extension"/>
+    <img src="https://img.shields.io/visual-studio-marketplace/v/oracle.suitecloud-vscode-extension.svg" alt="suitecloud-extension"/>
+</a>
+
```

Comment:
![image](https://user-images.githubusercontent.com/20229175/133988082-c24f7991-e2cd-421d-8ef8-682663cc752d.png)
Could it be that we see the badges in a different line because <a> tag starts in different line? Should lines 12 and 13 be together to have the same visualization as  we have for `SuiteCloud CLI for Node.js`?

Reply:
It's already fixed.